backend/corpora/dataset_processing/h5ad_data_file.py

```diff
@@ -273,13 +249,14 @@ def get_corpora_properties(self):
                 raise KeyError(f"missing Corpora schema field {key}")
             corpora_props[key] = self.anndata.uns[key]
 
-        for key in CorporaConstants.OPTIONAL_JSON_ENCODED_METADATA_FIELD:
+        for key in CorporaConstants.OPTIONAL_LIST_METADATA_FIELDS:
             if key not in self.anndata.uns:
                 continue
             try:
-                corpora_props[key] = json.loads(self.anndata.uns[key])
-            except json.JSONDecodeError:
-                raise json.JSONDecodeError(f"Corpora schema field {key} is expected to be a valid JSON string")
+                corpora_props[key] = self.anndata.uns[key].tolist()
```

Comment:
new support for lists encoded as ndarray

Reply:
Will do re: commit msg 👍
We didn't create any story for this piece, just for the db/api side https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1459